BootloaderCommonPkg/Library/UsbInitLib/UsbInitLib.c

```diff
@@ -37,12 +26,42 @@ RegisterUsbDevice (
   IN PEI_USB_IO_PPI   *UsbIoPpi
   )
 {
-  if (mUsbIoCount < ARRAY_SIZE (mUsbIoArray)) {
-    mUsbIoArray[mUsbIoCount++] = UsbIoPpi;
+  if (mUsbInit.UsbIoCount < ARRAY_SIZE (mUsbInit.UsbIoArray)) {
+    mUsbInit.UsbIoArray[mUsbInit.UsbIoCount++] = UsbIoPpi;
   }
```

Comment:
Better to handle error condition as well.

Reply:
sure, will add it.